proctor-common/src/main/java/com/indeed/proctor/common/Proctor.java

```diff
@@ -378,13 +378,7 @@ public ProctorResult determineTestGroups(
                     containsUnitlessAllocations ? VALUE_EXPRESSION_TRUE : VALUE_EXPRESSION_FALSE);
             final TestType testType = testChooser.getTestDefinition().getTestType();
             if (testChooser instanceof UnitlessTestChooser) {
-                identifier =
```

Comment:
Why the change here?

Reply:
After further discussion with steven, determined that null should be rejected and not able to get a result. 

This was causing a minor bug for an edge case where a non empty result was being returned. This fixes the bug and cleans up the logic. Making it more clear null is being rejected in the UnitlessTestChooser.

The edge case was in testDetermineTestGroupsWithNullIdentifier_UnitlessEnabled_falseAndTrueInRule